apps/dashboard/src/contract-ui/tabs/permissions/page.tsx

```diff
@@ -1,7 +1,9 @@
 "use client";
 
+import { InlineCode } from "@/components/ui/inline-code";
 import { useIsomorphicLayoutEffect } from "@/lib/useIsomorphicLayoutEffect";
-import { ButtonGroup, Code, Divider, Flex } from "@chakra-ui/react";
+import {} from "@chakra-ui/react";
```

Comment:
remove empty import

Reply:
oops. I trusted the linter to do its job!!